mmv1/templates/terraform/examples/storage_managed_folder_nested.tf.erb

```diff
@@ -0,0 +1,16 @@
+resource "google_storage_bucket" "bucket" {
+  name                        = "<%= ctx[:vars]['bucket_name'] %>"
+  location                    = "EU"
+  uniform_bucket_level_access = true
+  force_destroy               = true
+}
+
+resource "google_storage_managed_folder" "parent" {
```

Comment:
According to https://cloud.google.com/storage/docs/managed-folders#considerations you don't need to make the parent folder managed to make the nested folder managed.

Reply:
The idea here was to create a managed folder inside another managed folder, while the first example is just creating a managed folder at a given path. 

Let me know if this makes sense, otherwise we can keep the first example only.

Reply:
We should definitely have a test that creates a nested folder (and it should be the first example so that IAM tests use it as well). However, the nested folder doesn't need a "parent" folder. Users should be able to figure out how to do that if they want to; I don't think we need to document it separately, since it's not enforced by GCS.